prototypes/prototype.adoc

```diff
@@ -0,0 +1,18 @@
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
+:state: predraft
+:revremark: State: {state}
+
+:showtitle:
+:toc: left
+:numbered:
+:icons: font
+
+////
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Copyright 2017 <contribution>
+////
+
+# RFD <Number> <Title>
```

Comment:
I'd advocate for `=` in order to be idiomatic.  The token alias is nice, but we shouldn't rely on it. 

> When in Rome, do as the Romans do.